### PR TITLE
Fix #68: Hide FileMaterializer interface

### LINKED ISSUE #68: Hide file materializer interface
It is too complicated. A simple replacement configuration is probably enough.

### PR DESCRIPTION
**Pull request by gzm0**

### CODE REVIEW

File: seleniumJSEnv/src/main/scala/org/scalajs/jsenv/selenium/CustomFileMaterializer.scala [hunk 1]
```diff
@@ -1,55 +1,27 @@
...
-  /** Create a target file to write/copy to. Will also call
-    *  deleteOnExit on the file.
-    */
-  private def trgFile(name: String): File = {
-    val f = new File(storageDir, name)
-    f.deleteOnExit()
-    f
-  }
+private[selenium] class CustomFileMaterializer(contentDir: Path,
+    webRoot: URL) extends FileMaterializer {
```
Comment (sjrd) [inline seleniumJSEnv/src/main/scala/org/scalajs/jsenv/selenium/CustomFileMaterializer.scala]: Should this be renamed to `FileMaterializerInServer` or something more specific than "custom"?

File: seleniumJSEnv/src/main/scala/org/scalajs/jsenv/selenium/DefaultFileMaterializer.scala [hunk 2]
```diff
@@ -4,11 +4,8 @@ import org.scalajs.core.tools.io.VirtualTextFile
 import org.scalajs.jsenv.VirtualFileMaterializer
 import java.net.URL
 
-/** Materializes virtual files in a temporary directory and links to them
-  * via file://
-  */
-object DefaultFileMaterializer extends FileMaterializer {
-
+/** Materializes virtual files in a temp directory (uses file:// schema). */
+private[selenium] object DefaultFileMaterializer extends FileMaterializer {
```
Comment (sjrd) [inline seleniumJSEnv/src/main/scala/org/scalajs/jsenv/selenium/DefaultFileMaterializer.scala]: Similarly, maybe rename this to `FileMaterializerInTempDir` or `TempDirectoryFileMaterializer`?

File: seleniumJSEnv/src/main/scala/org/scalajs/jsenv/selenium/SeleniumJSEnv.scala [hunk 3]
```diff
@@ -67,8 +67,42 @@ object SeleniumJSEnv {
...
+     *
+     *  <pre>
+     *    jsSettings(
+     *      jsEnv := new SeleniumJSEnv(
+     *          org.openqa.selenium.remote.DesiredCapabilities.firefox(),
+     *          SeleniumJSEnv.Config()
+     *            .withMaterializeInServer(".tmp", "http://localhost:8080/")
+     *      )
+     *    )
+     *  </pre>
```
Comment (sjrd) [inline seleniumJSEnv/src/main/scala/org/scalajs/jsenv/selenium/SeleniumJSEnv.scala]: Scaladoc has dedicated syntax for code snippets: `{{{`..`}}}`-delimited blocks (usually without further increasing indentation inside them).

**Review by sjrd — CHANGES_REQUESTED** [review]

**Comment by gzm0** [comment]
Updated.

**Review by sjrd — APPROVED** [review]

**Pull request merged**